setup.py

```diff
@@ -0,0 +1,47 @@
+from setuptools import setup
+
+required = {'param':">=1.10.0",
+            'panel':">=0.10.0",
+            'pygridgen':">0.2",
+            'geopandas':">=0.8.1",
+            'holoviews':">=1.0.1"}
```

Comment:
Shouldn't these match what's in meta.yaml?

Reply:
Yes it should. Fixed the holoviews pinning in 297a480 (I checked the other pins and they are fine).